### PR TITLE
Add Bitwarden exception

### PR DESCRIPTION
**Pull request by brycx**
Bitwarden requires you to have a [premium](https://help.bitwarden.com/article/setup-two-step-login/) account to use hardware 2FA. I think it would be good to have an exception for this.

### CODE REVIEW

**Review by mxxcon — APPROVED** [review]

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
To me it looks like it's not just only for hardware 2FA.
I suggest you rephrase it to `"Premium account required for SMS, Phone call and hardware 2FA"` or something similar.
I'm not a native English speaker so the structuring might need some improvements ¯\_(ツ)_/¯

**Comment by mxxcon** [comment]
Carl, sms and phone call are properties of Duo's offering.
From what I can see you can't have them w/o Duo.
Perhaps change the wording further that Duo, Yubikey and FIDO U2F require paid plan?

**Comment by brycx** [comment]
Is this better?

**Review by Jawshy — APPROVED** [review]
👍

**Review by Carlgo11 — APPROVED** [review]
